### ISSUE TITLE
Expect response body can copy value or object like Chrome F12

### ISSUE THREAD

**Issue by totoropcbeta** [ProxymanApp/Proxyman#1002]
Proxyman
![image](https://user-images.githubusercontent.com/44626635/133886930-a41211d1-3c45-45e5-97fb-d002345b561d.png)

Chrome F12
![image](https://user-images.githubusercontent.com/44626635/133886971-d587cd00-19ea-4c3b-8f8f-e84c6b280590.png)

**Comment by NghiaTranUIT** [comment]
Thanks for your suggestion @totoropcbeta 👍 

Currently, you can copy the value by right-click on the row:

<img width="658" alt="Screen Shot 2021-09-18 at 19 02 50" src="https://user-images.githubusercontent.com/5878421/133888144-99435324-178b-4c0e-b3e5-ea06237b93cc.png">

I will support other copy functionality 🙌

**Comment by totoropcbeta** [comment]
emm,i tried it according to the way you supplied,but could not copy dictionary or array,like "makers",the result of the copy operation is "makers: Array(1 items)".so this could only copy value like "tagline" in your picture.

**Comment by NghiaTranUIT** [comment]
Yup, it's a limitation on the current build. Copy will copy the key and value, not the json object like Google Chrome 😄 

I'm working on this feature and will release in the next build 👍

**Comment by NghiaTranUIT** [comment]
Hey @totoropcbeta, please try this beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_2.33.0_Copy_Value_Object_in_JSONTreeView.dmg

You can now copy a Value or JSON node like Google Chrome 👍 

<img width="692" alt="Screen Shot 2021-09-26 at 08 54 47" src="https://user-images.githubusercontent.com/5878421/134792415-49a3f248-4dc4-4b6d-b218-6369817577e6.png">
